Library/Formula/gcovr.rb

```diff
@@ -0,0 +1,17 @@
+class Gcovr < Formula
+  homepage "http://gcovr.com/"
+  url "https://github.com/gcovr/gcovr/archive/3.2.tar.gz"
+  sha1 "97e44794969378db99cb77ee3c71b110dc634a8f"
+  head "https://github.com/gcovr/gcovr.git"
+
+  def install
+    prefix.install Dir["*"]
```

Comment:
I've preferred installing into `prefix` rather than `libexec` to avoid making the directory hierarchy of the formula one level deeper unnecessarily. I'll make the change as requested all the same. What's the motivation behind stuffing everything in libexec?